lib/cache.js

```diff
@@ -1,8 +1,31 @@
 'use strict'
 
+const Winston = require('winston')
 const LevelUp = require('level')
+const Sublevel = require('level-sublevel')
 const Path = require('path')
 
-const path = Path.join(__dirname, '..', 'data')
+const path = Path.join(__dirname, '..', 'cache')
```

Comment:
https://github.com/jembi/facility-cache/pull/1/files#diff-910eb6f57886ca16c136101fb1699231R51 and https://github.com/jembi/facility-cache/pull/1/files#diff-ff2899c03c9e8e31a9b69d7621e08c83R106 but reference the data directory.